src/components/DatePicker/DatePicker.st.css

```diff
@@ -0,0 +1,581 @@
+    */
+    WeekdayTextColor: --overridable;
+    /*
+        The font of a specific date. The font size is 14px and could not be overridden.
+        @default {theme: 'Body-M', lineHeight: '20px'}
+    */
+    DayFont: --overridable;
+    /*
+        The font color of a specific day.
+        @default color-5
+    */
+    DayColor: --overridable;
+    /*
+        The font color of the selected day.
+        @default color-1
+    */
+    SelectedDayColor: --overridable;
+    /*
+        The background color of the selected day.
+        @default color-color-8
+    */
+    SelectedDayBGColor: --overridable;
+    /*
+        The border radius of the selected day.
+        @default 50%
+    */
+    SelectedDayBorderRadius: --overridable;
+    /*
+        The background color of a disabled day. Disabled day is a date that are not selectable.
+        @default color-3
+    */
+    DisabledDayColor: --overridable;
+}
+
+/* Overrides - deceleration of overridable parameters by the consumer */
+:vars {
+    CurrentBackgroundColor: color(fallback(value(BackgroundColor), value(DefaultBackgroundColor)));
+
+    CurrentNavigationArrowsColor: color(fallback(value(NavigationArrowsColor), value(DefaultNavigationArrowsColor)));
+    CurrentMonthAndYearCaptionFont: fallback(font(value(MonthAndYearCaptionFont)), font(value(DefaultMonthAndYearCaptionFont)));
+    CurrentMonthAndYearCaptionColor: color(fallback(value(MonthAndYearCaptionColor), value(DefaultMonthAndYearCaptionColor)));
+    DropdownOptionSelectedAndHoverBGColor: applyOpacity(value(CurrentDayColor), 0.06);
+
+    CurrentWeekdayFont: fallback(font(value(WeekdayFont)), font(value(DefaultWeekdayFont)));
+    CurrentWeekdayTextColor: color(fallback(value(WeekdayTextColor), value(DefaultWeekdayTextColor)));
+
+    CurrentDayFont: fallback(font(value(DayFont)), font(value(DefaultDayFont)));
+    CurrentDayColor: color(fallback(value(DayColor), value(DeafultDayColor)));
+    CurrentDayBGColorHover: applyOpacity(value(CurrentSelectedDayBGColor), 0.1);
+    CurrentSelectedDayColor: color(fallback(value(SelectedDayColor), value(DefaultSelectedDayColor)));
+    CurrentSelectedDayBGColor: color(fallback(value(SelectedDayBGColor), value(DefaultSelectedDayBGColor)));
+    CurrentSelectedDayBorderRadius: fallback(value(SelectedDayBorderRadius),value(DefaultSelectedDayBorderRadius));
+    CurrentDisabledDayColor: color(fallback(value(DisabledDayColor), value(DefaultDisabledDayColor)));
+}
+
+/* The component's css without the overrideStyleParams */
+
+.root {
+}
+
+.root .popoverContent {
+    border-radius: 0;
+}
+
+.calendar {
+    -st-extends: WSRCalendar;
+}
+
+.calendar::month {
+    width: value(calendarWidth);
+    height: value(calendarHeight);
+}
+
+/* calendar header */
+
+.calendar::header {
+    padding: 28px value(sidePadding) 0;
+    background-color: value(CurrentBackgroundColor);
+}
+
+.calendar::header::arrow {
+    color: value(CurrentNavigationArrowsColor);
+    width: 24px;
+    height: 24px;
```

Comment:
If font will be overridable, we might need to have the arrow dimensions be set as `em`'s